server/routes/contacts/add/addContactFlowControl.ts

```diff
@@ -0,0 +1,80 @@
+import AddContactJourney = journeys.AddContactJourney
+import { Page } from '../../../services/auditService'
+
+interface AddContactNavigation {
+  backLink?: string
+}
+
+function navigationForAddContactJourney(currentPage: Page, journey: journeys.AddContactJourney): AddContactNavigation {
```

Comment:
These two functions are the actual change, most everything else is a result of renames.